libs/full/CMakeLists.txt

```diff
@@ -51,7 +52,7 @@ set(_hpx_full_modules
 # cmake-format: on
 
 hpx_info("")
-hpx_info("Configuring libhpx modules:")
+hpx_info("Configuring libhpx_full modules:")
```

Comment:
Enough with me just saying I'll fix this... I opened #5206 and (force-)pushed a change to revert these on this PR.